bin/tasks/gitInfo.test.js

```diff
@@ -9,10 +9,10 @@ const log = { debug: jest.fn() };
 
 describe('setGitInfo', () => {
   it('sets the git info on context', async () => {
-    getCommitAndBranch.mockReturnValue({ commit: '123asdf', branch: 'something' });
-    getBaselineCommits.mockReturnValue(['asd2344']);
-    getVersion.mockReturnValue('Git v1.0.0');
-    getSlug.mockReturnValue('user/repo');
+    getCommitAndBranch.mockReturnValue(Promise.resolve({ commit: '123asdf', branch: 'something' }));
```

Comment:
can you use `mockResolvedValue` for this?

Reply:
Nice one 👍